<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControllerDetails</class>
 <widget class="QDialog" name="ControllerDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icons/qutescoop.png</normaloff>:/icons/qutescoop.png</iconset>
  </property>
  <layout class="QVBoxLayout" stretch="0,1,0">
   <property name="margin" stdset="0">
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Controller</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="margin" stdset="0">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="lblControllerInfo">
        <property name="text">
         <string>Donald Fauntleroy Duck (C3)
On IVANIT for 01:23, IvAc 1.2.3</string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="detailsLayout">
        <item>
         <widget class="QLabel" name="lblStationInformatoin">
          <property name="text">
           <string>APP on frequency 123.45</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pbAirportDetails">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>LOWW (Vienna, AT)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbAtis">
     <property name="title">
      <string>ATIS</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin" stdset="0">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="lblAtis">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomLayout">
     <item>
      <widget class="QPushButton" name="buttonShowOnMap">
       <property name="text">
        <string>show on &amp;map</string>
       </property>
       <property name="shortcut">
        <string>S</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAddFriend">
       <property name="text">
        <string>add friend</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonShowOnMap</sender>
   <signal>clicked()</signal>
   <receiver>ControllerDetails</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>ControllerDetails</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
